docs/notebooks/flax_intro.ipynb

```diff
@@ -489,7 +489,7 @@
         "Model is a thin abstraction around a Module and a set of parameters.\n",
```

Comment:
Did you update the docs manually or from within Colab? I ask because it's easy to accidentally make these not build. If you edited them manually, please follow the instructions in docs/README.md to re-build the docs and make sure that the Flax Intro still appears and looks correct.

Reply:
I see.
I compiled the docs to make sure everything is fine.